Import failure with ipopt 3.13.x packages

<!--
Thanks for reporting your issue.
Please fill out the sections below.
-->
Issue:
Example: https://travis-ci.org/github/pycalphad/pycalphad/jobs/672180955
```
    import ipopt
../../../miniconda2/envs/condaenv/lib/python3.7/site-packages/ipopt/__init__.py:43: in <module>
    from cyipopt import *
E   ImportError: libipopt.so.1: cannot open shared object file: No such file or directory
```
<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/travis/miniconda2/envs/condaenv:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
alabaster                 0.7.12                     py_0    conda-forge
attrs                     19.3.0                     py_0    conda-forge
babel                     2.8.0                      py_0    conda-forge
backcall                  0.1.0                      py_0    conda-forge
bokeh                     2.0.1            py37hc8dfbb8_0    conda-forge
ca-certificates           2020.4.5.1           hecc5488_0    conda-forge
certifi                   2020.4.5.1       py37hc8dfbb8_0    conda-forge
cffi                      1.14.0           py37hd463f26_0    conda-forge
chardet                   3.0.4           py37hc8dfbb8_1006    conda-forge
click                     7.1.1              pyh8c360ce_0    conda-forge
cloudpickle               1.3.0                      py_0    conda-forge
coverage                  4.5.4            py37h516909a_0    conda-forge
coveralls                 2.0.0                      py_0    conda-forge
cryptography              2.8              py37hb09aad4_2    conda-forge
cycler                    0.10.0                     py_2    conda-forge
cyipopt                   0.1.9           py37hc1659b7_1000    conda-forge
cython                    0.29.16          py37h3340039_0    conda-forge
cytoolz                   0.10.1           py37h516909a_0    conda-forge
dask                      2.14.0                     py_0    conda-forge
dask-core                 2.14.0                     py_0    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
dill                      0.3.1.1          py37hc8dfbb8_1    conda-forge
distributed               2.14.0           py37hc8dfbb8_0    conda-forge
docopt                    0.6.2                      py_1    conda-forge
docutils                  0.16             py37hc8dfbb8_1    conda-forge
expat                     2.2.9                he1b5a44_2    conda-forge
fastcache                 1.1.0            py37h8f50634_1    conda-forge
fontconfig                2.13.1            he4413a7_1000    conda-forge
freetype                  2.10.1               he06d7ca_0    conda-forge
fsspec                    0.7.1                      py_0    conda-forge
future                    0.18.2           py37hc8dfbb8_1    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
gf2x                      1.2               h5586051_1003    conda-forge
glib                      2.58.3          py37he00f558_1003    conda-forge
gmp                       6.2.0                he1b5a44_2    conda-forge
gmpy2                     2.1.0b1          py37h04dde30_0    conda-forge
gst-plugins-base          1.14.5               h0935bb2_2    conda-forge
gstreamer                 1.14.5               h36ae1b5_2    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
icu                       58.2              hf484d3e_1000    conda-forge
idna                      2.9                        py_1    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib-metadata        1.6.0            py37hc8dfbb8_0    conda-forge
importlib_metadata        1.6.0                         0    conda-forge
ipopt                     3.13.1               hc6cf775_0    conda-forge
ipython                   7.13.0           py37hc8dfbb8_2    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.16.0           py37hc8dfbb8_1    conda-forge
jinja2                    2.11.1                     py_0    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
kiwisolver                1.2.0            py37h99015e2_0    conda-forge
ld_impl_linux-64          2.33.1               h53a641e_7  
libblas                   3.8.0               14_openblas    conda-forge
libcblas                  3.8.0               14_openblas    conda-forge
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1                hd88cf55_4  
libflint                  2.5.2                heb735ee_2    conda-forge
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_5    conda-forge
libiconv                  1.15              h516909a_1006    conda-forge
liblapack                 3.8.0               14_openblas    conda-forge
libopenblas               0.3.7                h5ec1e0e_6    conda-forge
libpng                    1.6.37               hed695b0_1    conda-forge
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.1.0                hc7e4089_6    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libwebp-base              1.1.0                h516909a_3    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.9                hea5a465_1  
locket                    0.2.0                      py_2    conda-forge
lz4-c                     1.8.3             he1b5a44_1001    conda-forge
markupsafe                1.1.1            py37h8f50634_1    conda-forge
matplotlib                3.1.3                    py37_0  
matplotlib-base           3.1.3            py37hef1b27d_0  
more-itertools            8.2.0                      py_0    conda-forge
mpc                       1.1.0             h04dde30_1006    conda-forge
mpfr                      4.0.2                he80fd80_0    conda-forge
mpmath                    1.1.0                      py_0    conda-forge
msgpack-python            1.0.0            py37h99015e2_1    conda-forge
ncurses                   6.2                  he6710b0_0  
ntl                       10.3.0                        1    conda-forge
numpy                     1.18.1           py37h8960a57_1    conda-forge
olefile                   0.46                       py_0    conda-forge
openssl                   1.1.1f               h516909a_0    conda-forge
packaging                 20.1                       py_0    conda-forge
pandas                    1.0.3            py37h0da4684_0    conda-forge
parso                     0.6.2                      py_0    conda-forge
partd                     1.1.0                      py_0    conda-forge
pcre                      8.44                 he1b5a44_0    conda-forge
pexpect                   4.8.0            py37hc8dfbb8_1    conda-forge
pickleshare               0.7.5           py37hc8dfbb8_1001    conda-forge
pillow                    7.0.0            py37hb39fc2d_0  
pip                       20.0.2                   py37_1  
pluggy                    0.12.0                     py_0    conda-forge
prompt-toolkit            3.0.5                      py_0    conda-forge
psutil                    5.7.0            py37h8f50634_1    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
py                        1.8.1                      py_0    conda-forge
pycalphad                 0.8.3+3.g7307b13           dev_0    <develop>
pycparser                 2.20                       py_0    conda-forge
pygments                  2.6.1                      py_0    conda-forge
pyopenssl                 19.1.0                     py_1    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.9.2            py37hcca6a23_4    conda-forge
pysocks                   1.7.1            py37hc8dfbb8_1    conda-forge
pytest                    5.4.1            py37hc8dfbb8_0    conda-forge
pytest-cov                2.8.1                      py_0    conda-forge
python                    3.7.7           hcf32534_0_cpython  
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
pytz                      2019.3                     py_0    conda-forge
pyyaml                    5.3.1            py37h8f50634_0    conda-forge
qt                        5.9.7                h52cfd70_2    conda-forge
readline                  8.0                  h7b6447c_0  
requests                  2.23.0             pyh8c360ce_2    conda-forge
scipy                     1.4.1            py37ha3d9a3c_2    conda-forge
setuptools                46.1.3                   py37_0  
sip                       4.19.8           py37hf484d3e_0  
six                       1.14.0                     py_1    conda-forge
snowballstemmer           2.0.0                      py_0    conda-forge
sortedcontainers          2.1.0                      py_0    conda-forge
sphinx                    3.0.0                      py_0    conda-forge
sphinx_rtd_theme          0.4.3                      py_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.3                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.4                      py_0    conda-forge
sqlite                    3.31.1               h7b6447c_0  
symengine                 0.6.0                    pypi_0    pypi
sympy                     1.5.1            py37hc8dfbb8_3    conda-forge
tblib                     1.6.0                      py_0    conda-forge
tinydb                    3.15.2                   pypi_0    pypi
tk                        8.6.8                hbc83047_0  
toolz                     0.10.0                     py_0    conda-forge
tornado                   6.0.4            py37h8f50634_1    conda-forge
traitlets                 4.3.3            py37hc8dfbb8_1    conda-forge
typing_extensions         3.7.4.1          py37hc8dfbb8_3    conda-forge
urllib3                   1.25.7           py37hc8dfbb8_1    conda-forge
wcwidth                   0.1.9              pyh9f0ad1d_0    conda-forge
wheel                     0.34.2                   py37_0  
xarray                    0.15.1                     py_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xz                        5.2.4                h14c3975_4  
yaml                      0.2.2                h516909a_1    conda-forge
zict                      2.0.0                      py_0    conda-forge
zipp                      3.1.0                      py_0    conda-forge
zlib                      1.2.11               h7b6447c_3  
zstd                      1.4.4                h3b9ef0a_2    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):

conda 4.8.3

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -15,7 +15,7 @@ source:
 
 build:
   # TODO : Set the build number to 0 on the next cyipopt release.
-  number: 1002
+  number: 1003
   # TODO: Once ipopt is available for Windows in conda forge, remove the
   # following line.
   skip: True  # [win]
@@ -25,10 +25,12 @@ requirements:
   build:
     - pkg-config  # [osx]
     - {{ compiler('c') }}
+    - lapack
   host:
     - pip
     - python
-    - ipopt <3.13
+    - ipopt
+    - lapack
     - setuptools
     - cython
     - numpy
@@ -36,7 +38,7 @@ requirements:
     - six
   run:
     - python
-    - ipopt <3.13
+    - ipopt
     - {{ pin_compatible('numpy') }}
     - scipy
     - six

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -25,6 +25,7 @@ requirements:
   build:
     - pkg-config  # [osx]
     - {{ compiler('c') }}
+    # NOTE : The conda-forge ipopt vendors lapack, thus it is needed explicitly.
     - lapack
   host:
     - pip

